documentation/config_logging_advanced.markdown

```diff
@@ -69,7 +69,6 @@ To add JSON logging for HTTP requests:
 1.  Add the following Logback appender section to the `request-logging.xml` file:
 
     ``` xml
-    {% raw %}
```

Comment:
Yeah, without these `{% raw %}` liquid tags, the pages will fail during the docs site build. This is due to the xml containing liquid-like structures, such as `{%statusCode}`.